tests/e2e/controlplane/control_plane_test.go

```diff
@@ -232,6 +236,37 @@ spec:
 							ShouldNot(ContainSubstring("Reconciliation done"), "Istio Operator is continuously reconciling")
 						Success("Istio Operator stopped reconciling")
 					})
+
+					It("can be deployed bookinfo with sidecar injection", func(ctx SpecContext) {
```

Comment:
Hm, I still don't think "it can be deployed bookinfo with sidecar injection" is a proper english sentence.

Also, shouldn't this whole test be in its own separate `When("bookinfo is deployed) ...` ?

Reply:
Yeah, make sense to be in a different node. Let me check and do some changes

Reply:
Updated